cmd/deploy/deploy.go

```diff
@@ -460,6 +464,9 @@ func setDeployOptionsValuesFromManifest(deployOptions *Options, cwd string) {
 				deployOptions.servicesToDeploy = append(deployOptions.servicesToDeploy, service)
 			}
 		}
+		if len(deployOptions.Manifest.Deploy.ComposeSection.ComposesInfo) > 0 {
+			deployOptions.Manifest.Deploy.ComposeSection.ComposesInfo[0].ServicesToDeploy = stack.AddDependentServicesIfNotPresent(ctx, deployOptions.Manifest.Deploy.ComposeSection.Stack, deployOptions.servicesToDeploy, c)
```

Comment:
It does, We have a merged stack and need a list of services to deploy which we get from getting all the services to deploy on each of the deploy manifest we are not checking which services comes from which stack.
I thought to set the services on the correct stack but I couldn't do it because it needs to read again all the stacks and it doesn't provide any user experience